src/state/settings.js

```diff
@@ -37,6 +41,9 @@ const settings = {
 		setShowVirtualLevels(state, val) {
 			state.showVirtualLevels = val;
 		},
+		showFullValues(state, val) {

```

Comment:
This is going to have issues being named the same as the getter.

```suggestion
		setShowFullValues(state, val) {

```